lms/static/js/sysadmin/mgmt_commands.js

```diff
@@ -0,0 +1,311 @@
+        var command_key = $(e.target).val();
+        loadCommand(command_key);
+    }
+
+
+    function updateElementWithContent($element, content){
+        var paragraph = $(document.createElement('p'));
+        paragraph.html(content.replace(/(?:\r\n|\r|\n)/g, '<br />'));
+        $element.append(paragraph)
+    }
+
+    function submitForm(e){
+        var button = $(e.target);
+        button.prop('disabled',true);
+        var form = $('#command-form');
+
+        $.ajax({
+            url: '/sysadmin/execute_command/',
+            type: 'POST',
+            data: form.serialize(),
+            dataType: 'json',
+            success: function(data) {
+
+                $errorBox = $('.mgmt-commands-body .error');
+                $stdErrLogBox = $('.mgmt-commands-body .stderr');
+                $stdOutLogBox = $('.mgmt-commands-body .stdout');
+
+                $errorBox.html('<h3>Errors:</h3>');
+                $stdErrLogBox.html('<h3>Std Error:</h3>');
+                $stdOutLogBox.html('<h3>Std Out:</h3>');
+
+                if (data.error != null){
+                    updateElementWithContent($errorBox, data.error);
+
+                }
+                if (data.stderr != null){
+                    updateElementWithContent($stdErrLogBox, data.stderr);
+                }
+                if (data.stdout != null){
+                    updateElementWithContent($stdOutLogBox, data.stdout);
+                }
+                button.prop('disabled', false);
+            },
+            error: function(std_ajax_err) {
+              console.log('Management Command failed to execute');
+            }
+        });
+    }
+
+    function generateInputRow(arg, type){
+        var $inputRow = $(document.createElement('div'));
+        $inputRow.addClass('form-actions')
+        var label = $(document.createElement('label'));
+        label.html(arg.display_name);
+        if (arg.required){
+            label.append('<span>*</span>')
+        }
+
+        var input = $(document.createElement('input'));
+
+        if (type == 'kwarg'){
+
+            label.attr('for', arg.argument);
+            input.attr('type', 'text');
+            input.attr('name', arg.argument);
+            $inputRow.append(label);
+            $inputRow.append(input);
+        }
+        else if (type == 'kwflag'){
+            label.attr('for', 'kwflags');
+            label.addClass('checkbox-label');
+            input.attr('type', 'checkbox');
+            input.attr('name', 'kwflags');
+            input.val(arg.argument);
+            $inputRow.append(input);
+            $inputRow.append(label);
+        }
+        else if (type === 'arg'){
+            label.attr('for', 'args');
+            input.attr('type', 'text');
+            input.attr('name', 'args');
+            $inputRow.append(label);
+            $inputRow.append(input);
+        }
+
+
+        return $inputRow;
+    }
+
+    function loadCommand(command_key){
```

Comment:
why not default `command_key=null` here? This would allow the call below `loadCommand(0);` to be empty and better: `loadCommand();`